build-system/tasks/check-owners.js

```diff
@@ -22,28 +22,66 @@
 
 'use strict';
 
+const argv = require('minimist')(process.argv.slice(2));
+const BBPromise = require('bluebird');
+const requestPost = BBPromise.promisify(require('request').post);
 const fs = require('fs-extra');
 const globby = require('globby');
 const JSON5 = require('json5');
 const log = require('fancy-log');
-const {cyan, red, green} = require('ansi-colors');
+const {getFilesToCheck} = require('../common/utils');
+const {cyan, red, yellow, green} = require('ansi-colors');
 const {isTravisBuild} = require('../common/travis');
 
+const OWNERS_SYNTAX_CHECK_URI = 'http://ampproject-owners-bot.appspot.com/v0/syntax';
+
 /**
  * Checks OWNERS files for correctness using the owners bot API.
  * The cumulative result is returned to the `gulp` process via process.exitCode
  * so that all OWNERS files can be checked / fixed.
  */
 async function checkOwners() {
-  const filesToCheck = globby.sync(['**/OWNERS']);
-  filesToCheck.forEach(checkFile);
+  if (!isValidUsage()) {
+    return;
+  }
+  const filesToCheck = getFilesToCheck('**/OWNERS');
+  for (const file of filesToCheck) {
+    await checkFile(file);
+  }
+}
+
+/**
+ * Checks if the correct arguments were passed in
+ *
+ * @return {boolean}
+ */
+function isValidUsage() {
```

Comment:
This is almost identical to the version in `check-links`. Can you move it to `utils.js`?